userland/libtock/ble.c

```diff
@@ -1,43 +1,148 @@
+/*
+ * BLE setup functions
+ */
+
 #include "ble.h"
+#include "tock.h"
+#include <stdbool.h>
+#include <stdint.h>
 #include <stdio.h>
 #include <string.h>
 
-int ble_adv_set_txpower(BLE_TX_Power_t power) {
-  return command(DRIVER_RADIO, BLE_CFG_TX_POWER, power);
+#define MAX_SIZE 31
+
+
+/*******************************************************************************
+ *   INTERNAL BLE HELPER FUNCTION Prototypes
+ *
+ *   s_   - static (file scope)
+ ******************************************************************************/
```

Comment:
It would be useful to have comments for what these functions do, even though they are internal helper functions.